sklearn_numba_dpex/common/random.py

```diff
@@ -33,7 +33,7 @@ def get_random_raw(states):
 
     Note, this always uses and updates state states[0].
     """
-    result = dpt.empty(sh=(1,), dtype=np.uint64, device=states.device)
```

Comment:
This prepares compatibility with latest versions of `dpctl`.

Straight removing `sh=` rather than replacing with `shape=` enables compatibility with all version before and after the breaking changes.

Reply:
Note that in this case, passing 1 instead of `(1,)` might also work (I think it does for numpy).